src/components/AppHeader.vue

```diff
@@ -27,11 +20,19 @@
 									>All DAOs</a-button
 								>
 							</router-link>
+							<a-button
+								class="menu_option"
+								:type="isLinkActive(NavigationKey.CREATE_DAO)"
+								@click="() => handleMenuClick(NavigationKey.CREATE_DAO)"
+								>Create DAO</a-button
+							>
+
 							<router-link :to="{ path: EndPoint.ADD_PROPOSAL }">
 								<a-button
 									class="menu_option"
 									:type="isLinkActive(NavigationKey.ADD_PROPOSAL)"
 									@click="() => handleMenuClick(NavigationKey.ADD_PROPOSAL)"
+									:disabled="!daoID || showOptIn"
```

Comment:
I see `!daoID || showOptIn` repeat few times. Why you don't create variable to store it in data?

Reply:
yes, will update